### DISCUSSION TITLE
Version-Info for compiling deepmd.2.0.0.b3

### DISCUSSION THREAD

**Discussion by AvMach** [deepmodeling/deepmd-kit#902] [Q&A]
Hello,

I want to install deepmd for the usage with an Nvidia RTX 3080 which requires CUDA 11.1 . I now discovered the new 2.0.0.b3 version with the code for cuda 11.1. I was wondering if you could tell me the version information for the dependent software like you have on the deepmd.org page you can see below.
![Screenshot 2021-07-30 124557](https://user-images.githubusercontent.com/80164998/127642164-04304a6a-1c7d-441a-89fc-b7854f60f453.png)

**Comment by njzjz** [comment]
We have prepared some pre-built binary packages on the release page. If you need to build yourself, I think you need to build tensorflow>=2.4 where CUDA 11 is supported.
